oc.mk

```diff
@@ -39,8 +39,8 @@ define oc_validate
 endef
 
 define oc_lint
-	@@shopt -s globstar nullglob; \
-		for FILE in openshift/**/*.yml; do \
+	@@shopt -s nullglob extglob; \
+		for FILE in $$(ls openshift/*/!(secret)/*.yml); do \
```

Comment:
we're working on getting a reserved namespace for validation so this makes sense until we're running against one with nothing in it (because we need read permission on an object to lint it)